en/docs/InstallAndSetup/UpgradingWSO2APIManager/upgrading-from-210-to-300.md

```diff
@@ -1,303 +1,1908 @@
+        "isCascadeDeleteEnabled": true,
+        "initializeNewClaimManager": true
+        }
+    }
+    ```
 
-`/sequences/_production_key_error_.xml`
+5.  Copy the relevant JDBC driver to the `<API-M_3.0.0_HOME>/repository/components/lib` folder.
 
-`/sequences/_resource_mismatch_handler_.xml`
+    !!! info
+        In API-M 3.0.0, you do not need to configure the registry configurations as you did in the `<OLD_API-M_HOME>/repository/conf/registry.xml` file and the user database configurations as you did in in the `<OLD_API-M_HOME>/repository/conf/user-mgt.xml` file, as those configurations have been handled internally.
 
-`/sequences/_sandbox_key_error_.xml`
+6.  Move all your Synapse configurations to API-M 3.0.0 pack.
+    -   Move your Synapse super tenant configurations.
+        Copy the contents in the `<OLD_API-M_HOME>/repository/deployment/server/synapse-configs/default` directory and replace the contents in the `<API-M_3.0.0_HOME>/repository/deployment/server/synapse-configs/default` directory with the copied contents.
+    -   Move all your tenant Synapse configurations.
+        Copy the contents in the `<OLD_API-M_HOME>/repository/tenants` directory and replace the contents in the `<API-M_3.0.0_HOME>/repository/tenants` directory with the copied contents.
 
-`/sequences/_throttle_out_handler_.xml`
+    !!! warning
+        When moving the Synapse configurations, **do not replace** the following set of files as they contain some modification in API-M 3.0.0 version.
 
-`/sequences/_token_fault_.xml`
+        -   /api/_RevokeAPI_.xml
+        -   /sequences/_cors_request_handler_.xml
+        -   /sequences/main.xml
+        -   proxy-services/WorkflowCallbackService.xml
 
-`/proxy-services/WorkflowCallbackService.xml`
+    !!! attention 
+        If you are working with a **clustered/distributed API Manager setup**, follow this step on the **Gateway** node.
 
+7.  Move all your Execution plans from `<API-M_2.6.0_HOME>/repository/deployment/server/executionplans` directory to `<API-M_3.0.0_HOME>/repository/deployment/server/executionplans` directory.
 
-6.  Move all your tenant Synapse configurations by copying the contents in the `<CURRENT_API-M_HOME>/repository/tenants` directory and adding the copied contents to the `<API-M_2.6.0_HOME>/repository/tenants` directory.
+    !!! note
+        If you are working with a **clustered/distributed API Manager setup**, follow this step on the **Traffic Manager** node.
 
-        !!! warning
-    **NOTE:** Copy the files listed below from the `<API-M_2.6.0_HOME>/repository/deployment/server/synapse-configs/default/sequences` directory and replace the corresponding files in the `<API-M_2.6.0_HOME>/repository/tenants/<tenant-id>/synapse-configs/default/sequences` directory.
+8.  If you manually added any custom OSGI bundles to the `<API-M_3.0.0_HOME>/repository/components/dropins` directory, copy those to the `<API-M_3.0.0_HOME>/repository/components/dropins` directory. 
```

Comment:
Change the API-M version as 2.1.0 in the path 
``` suggestion
8.  If you manually added any custom OSGI bundles to the `<API-M_2.1.0_HOME>/repository/components/dropins` directory, copy those to the `<API-M_3.0.0_HOME>/repository/components/dropins` directory. 
```

Reply:
Will fix in the next PR